go/cmd/vtctldclient/internal/command/vschemas.go

```diff
@@ -141,7 +148,33 @@ func commandGetVSchema(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var validateVSchemaOptions = struct {
+	Shards        []string
+	ExcludeTables []string
+	IncludeViews  bool
+}{}
+
+func commandValidateVSchema(cmd *cobra.Command, args []string) error {
+	keyspace := cmd.Flags().Arg(0)
+
+	cli.FinishedParsing(cmd)
```

Comment:
same comment here. `cli.FinishedParsing` should happen _as soon as_ we are done validating input (i.e. right after the last time we might return an error from invalid input). since none of these functions have to do any input validation, it should be the very first line.